app/views/govuk_publishing_components/components/docs/title.yml

```diff
@@ -78,4 +78,15 @@ examples:
       margin_bottom: 0
     context:
       dark_background: true
-
+  using_design_system_template:
```

Comment:
Can you update the other example of margin in this file from 'With margin' to 'With margin bottom' ?